be/src/http/action/stream_load.cpp

```diff
@@ -98,6 +98,40 @@ static TFileFormatType::type parse_format(const std::string& format_str,
         if (compress_type.empty()) {
             format_type = TFileFormatType::FORMAT_JSON;
         }
+    } else if (boost::iequals(format_str, "CSV_WITH_NAMES")) {
```

Comment:
looks these codes the same as boost::iequals(format_str, "CSV")？

Reply:
It has been optimized